records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+---
+layout: contact
+title: Scala Center Activity Report for 2022 Q4
+---
+
+Scala Center team: Quentin Bernet, 100%; Sébastien Doeraene, 100%; Darja Jovanovic, 100%; Anatolii Kmetiuk, 100%;
+Guillaume Martres, 100%; Adrien Piquerez, 100%; Julien Richard-Foy, 80%; Jamie Thompson, 100%.
+VirtusLab team: Jędrzej Rochala, 100%. Lunatech team: Chris Kipp, 33%.
+
+External contractors, part-time: Adam Goodman, leadership and
+governance expert.
+
+## At a Glance
+{: .no_toc}
+
+* Table of Contents
+{:toc}
+
+## Tooling
+
+### Debugger in Metals
+
+For Scala 2 and Scala 3.
+
+We shipped a new major version of the Scala Debug Adapter in Metals that improves the debugging experience in Scala 2 and Scala 3. In this new version, we improved the debugging steps, we added support for conditional breakpoints, logpoints and more features. See a full list of features in the release notes of the Scala Debug Adapter [3.0.1](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1) and [3.0.3](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.3).
+
+### BSP defaults for Metals
+
+An area of improvement we've identified with Metals and the usage of the Build
+Server Protocol (BSP) in general is that it provides many ways to get started
+with a Scala project. We'd like to minimize this and revisit the default choices
+that Metals makes of using Bloop no matter the build tool. The aim is to make
+the experience a newcomer has smoother, abstracting away things like Bloop and
+BSP, while still allowing for a powerful user experience for advanced users.
+We're still in the experiment and research stage for this which is leading to
+creating various issues around the ecosystem. However, the discussion and main
+work can be tracked in [metals#4505](https://github.com/scalameta/metals/discussions/4505#discussioncomment-4571546).
```

Comment:
Should we maybe also link to https://contributors.scala-lang.org/t/revisiting-the-default-build-server-for-metals-roadmap/6054?

Reply:
Done in fce926e36f94706c5bedd331b59ffca522fecc59